playground/mocks/data/idp/idx/authenticator-verification-duo.json

```diff
@@ -129,7 +129,7 @@
             "displayName": "Duo Security",
             "methods": [
                 {
-                    "type": "idp"
+                    "type": "duo"
```

Comment:
this is how the response is now. Not sure if `idp` before was a typo or the response actually changed